buildenv/jenkins/README.md

```diff
@@ -100,614 +96,145 @@ Having a complicated regex in the pull request trigger is what allows us to laun
-    - Description:
-        - Runs extended test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Test-Extended-JDK9-linux_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Extended-JDK9-linux_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Extended-JDK9-linux_x86-64_cmprssptrs)
-    - Description:
-        - Runs extended test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
+#### Pipelines
 
-- Test-Extended-JDK10-linux_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Extended-JDK10-linux_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Extended-JDK10-linux_x86-64_cmprssptrs)
-    - Description:
-        - Runs extended test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
+Pipelines for all platforms and versions are available [**here**](https://ci.eclipse.org/openj9/view/Pipelines/).
 
-- Test-Extended-JDK8-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Extended-JDK8-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Extended-JDK8-win_x86-64_cmprssptrs)
+- Any platform and version pipeline:
+    - Name: Pipeline-Build-Test-JDK`<version>`-`<platform>`
     - Description:
-        - Runs extended test suite against the SDK and test material that is passed as parameters
+        - Compile Eclipse OpenJ9 on `<platform>` for Extensions OpenJDK`<version>` and run sanity & extended test suites
+        - Triggers:
+            - Build-JDK`<version>`-`<platform>`
+            - Test-`<sanity|extended>.<functional|system>`-JDK`<version>`-`<platform>`
     - Trigger:
-        - This job is used in other pipelines but can be launched manually
+        - build periodically, @midnight
 
-- Test-Extended-JDK9-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Extended-JDK9-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Extended-JDK9-win_x86-64_cmprssptrs)
+- Pipeline-OMR-Acceptance
+    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Pipeline-OMR-Acceptance)](https://ci.eclipse.org/openj9/job/Pipeline-OMR-Acceptance)
     - Description:
-        - Runs extended test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
+        - Compile and run sanity tests against new OMR content
+        - Triggers:
+            - Build-JDK`<version>`-`<platform>` and Test-Sanity-JDK`<version>`-`<platform>` across all platforms and JDK versions 8, 10
```

Comment:
This is version 11 now instead of 10. In the future it may include 12 or newer versions. Probably want to avoid specifying here the exact versions since it can change.

Reply:
@pshipton - updated line - removed 8, 10
new line:
`- Build-JDK`<version>`-`<platform>` and Test-Sanity-JDK`<version>`-`<platform>` across all platforms and versions`